Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/ComponentAdapter.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/UI/DocumentPropertyEditor/ComponentAdapter.h>
+#include <QtCore/QTimer>
+
+namespace AZ::DocumentPropertyEditor
+{
+    ComponentAdapter::ComponentAdapter()
```

Comment:
maybe just "= default;" this in the declaration, unless you have plans to put things here.

Reply:
Thanks, fixed!